client/finality-grandpa/rpc/src/lib.rs

```diff
@@ -48,7 +48,7 @@ type FutureResult<T> =
 
 /// Provides RPC methods for interacting with GRANDPA.
 #[rpc]
-pub trait GrandpaApi<Notification, Hash> {
+pub trait GrandpaApi<Notification, Hash, N> {
```

Comment:
Let's name it `Number`, I've taken some fondness to descriptive type parameter names and it's more inline with the style of the existing parameters.